rotkehlchen/chain/ethereum/modules/makerdao/cache.py

```diff
@@ -129,7 +129,7 @@ def query_ilk_registry(
     - RemoteError if any of the remote queries fail
     """
     ilks_mapping = {}
-    ilk_registry = ethereum.contracts.contract('ILK_REGISTRY')
+    ilk_registry = ethereum.contracts.contract(string_to_evm_address('0x5a464C28D19848f44199D003BeF5ecc87d090F87'))  # noqa: E501
```

Comment:
I don't have this address in my globaldb. Where did you get it from?

Reply:
You can check globaldb on develop. It exists there
```bash
(venv) nebolax@Nebolax-PC:~/p/rotki/rotki$ sqlite3 rotkehlchen/data/global.db 
SQLite version 3.31.1 2020-01-27 19:55:54
Enter ".help" for usage hints.
sqlite> select * from contract_data where name="ILK_REGISTRY";
0x5a464C28D19848f44199D003BeF5ecc87d090F87|1|ILK_REGISTRY|67|12251871
sqlite> 
```